R/pkg.R

```diff
@@ -272,7 +278,8 @@ getPackageRecords <- function(pkgNames,
           lib.loc = lib.loc,
           missing.package = missing.package,
           check.lockfile = check.lockfile,
-          fallback.ok = fallback.ok
+          fallback.ok = fallback.ok,
+          .visited.packages
```

Comment:
nit: pass along as named argument here, just to be safe?

Reply:
thanks. i was rushing. :)